src/views/BusinessCase/Review/index.tsx

```diff
@@ -100,8 +100,9 @@ const Review = ({ formikProps }: ReviewProps) => {
           Alternatives analysis
         </h2>
       </div>
-      <div className="grid-container bg-base-lightest padding-top-2 padding-bottom-8">
+      <div className="bg-base-lightest padding-top-2 padding-bottom-8">
         <ResponsiveTabs
+          className="grid-container padding-x-0"
```

Comment:
I don't think we should add a `className` prop here. Instead we should wrap `ResponsiveTabs` in `<div className="grid-container>` because the `padding-x-0` causes the component to go edge-to-edge which isn't good.

We want to keep the padding from `grid-container` so that the component can have some space.

Reply:
If I'm not mistaken, the component _should_ go edge-to-edge. Here's a pic from the wireframe. Is this what you mean? 
![image](https://user-images.githubusercontent.com/7664177/82522853-7fc19900-9adf-11ea-8236-b6ea363e0f10.png)

Reply:
By which I mean, the grey background should go edge-to-edge, not the panel itself.

Reply:
Right, if you resize the screen smaller, you’ll see that the tabs panel has no room to breathe on the left and right.